editor/editor_file_system.cpp

```diff
@@ -1341,8 +1338,7 @@ void EditorFileSystem::_scan_fs_changes(EditorFileSystemDirectory *p_dir, ScanPr
 				ia.file = p_dir->files[i]->file;
 				scan_actions.push_back(ia);
 			}
-		} else if (ResourceCache::has(path)) { //test for potential reload
-
+		} else {
```

Comment:
This causes reload attempt for non-loaded Resources (I think?). Why this change?

Reply:
The problem that this fix is if a script is modified in an external editor without LSP (or just a git pull) and that script was never loaded, it will never be processed and that causes the documentation to not update and the file last modification date was never updated the EditorFileSystem.

Reply:
Non-loaded Resources shouldn't need reloading, only updating their last checked time 🤔

Reply:
One thing that is important to known is that the same check is done in `EditorNode::_resources_changed` to prevent resources that are not already loaded to reload. Also, I don't think it's possible to update du documentation without reloading the script. So, my comprehension is that for script, they will all be reloaded in `EditorFileSystem::_update_script_documentation` but for other resources, they should be reloaded only if they were already in the cache due to the condition in `EditorNode::_resources_changed`. The difference is that in the `resources_reload` signal we will also receive resources that were not already loaded.

I think, after thinking about it, I should put back the condition just to add the path in the `reloads` list. That way, only resources that have been already loaded will be present in the `resources_reload` signal and we still get the documentation updated. Sounds good?

Reply:
Yep.

Reply:
Done!